py_dtn7/bundle.py

```diff
@@ -3,508 +3,773 @@
 from __future__ import annotations
 
 from abc import ABC
-from datetime import datetime
-from enum import Enum
-from typing import Optional, Type, List
+from typing import Optional, List, Tuple
 
-import cbor2
+try:
+    from cbor2 import dumps, loads
+except ImportError:
+    from cbor import dumps, loads
```

Comment:
Consider differentiating the import with `if sys.implementation.name != 'micropython':`

Reply:
I agree, this is probably the best solution. Also look at my comment for your proposed modifications of __init__.py
https://github.com/teschmitt/py-dtn7/pull/4#discussion_r1150866335